files/default/vendor/chef-server/fetcher.rb

```diff
@@ -132,5 +132,14 @@ def chef_server_url
       m = %r{^#{@config['server']}/owners/(?<owner>[^/]+)/compliance/(?<id>[^/]+)/tar$}.match(@target)
       "#{m[:owner]}/#{m[:id]}"
     end
+
+    def raise_not_found
```

Comment:
This method doesn't add any value, as far as I can see - you could just call target_path directly on line 102.  Not a big deal, though.

Reply:
@clintoncwolfe Noted, thanks

Reply:
@clintoncwolfe Updated as per comment Please have a look.